wordpress-mobile/WordPress-iOS
# [Stats Widgets] Allow Today widget to expand/collapse

Fixes #12849 

This allows the Today widget to expand. When expanded, `Likes` and `Comments` are displayed.

The view now uses a `tableView` with custom cells to display the widget.
- If unconfigured, a `WidgetUnconfiguredCell` is displayed.
- If configured:
  - A `WidgetTwoColumnCell` is displayed for each row.
  - When the widget display mode is changed, the `Likes` and `Comments` row is inserted/deleted accordingly.
  - A `WidgetFooterView` displays the site url in the table footer.

To test:

---
- Start with a fresh install.
- Add the WP widget to the Today view.
- Verify the unconfigured view still looks like this.

![unconfigured_light](https://user-images.githubusercontent.com/1816888/68816804-cae48600-063c-11ea-98ae-8993d5c3c1bf.jpeg)

---
- Log into the app and go to Stats.
- Select 'Today' in the nav bar and select 'Use this site'.
- Return to the widget.
- Verify it now contains data for the selected site.
- Verify expanding/collapsing toggles Likes & Comments.

![expandy](https://user-images.githubusercontent.com/1816888/68817878-50b60080-0640-11ea-9600-2797554b1254.gif)

---
- Go back to the app and log out.
- Return to the widget.
- Verify the unconfigured view is displayed correctly.

---
Update release notes:
